client-tags/message-status.md

```diff
@@ -0,0 +1,83 @@
+    [recv] @label=ID;time=YYYY-MM-DDThh:mm:ss.sssZ;delivered :nick!ident@host TAGMSG target
+    [recv] @label=ID;time=YYYY-MM-DDThh:mm:ss.sssZ;read :nick!ident@host TAGMSG target
+
+Sender sends a `batch` (initially out of focus, brought into focus after batch is complete)
+
+    [send] :irc.host BATCH +ID target
+    [send] @batch=ID;draft/msgid=ID1;time=YYYY-MM-DDThh:mm:ss.sssZ :nick!ident@host PRIVMSG target :message
+    [recv] @label=ID1;time=YYYY-MM-DDThh:mm:ss.sssZ;delivered :nick!ident@host TAGMSG target
+    [send] @batch=ID;draft/msgid=ID2;time=YYYY-MM-DDThh:mm:ss.sssZ :nick!ident@host PRIVMSG target :message
+    [recv] @label=ID2;time=YYYY-MM-DDThh:mm:ss.sssZ;delivered :nick!ident@host TAGMSG target
+    [send] @batch=ID;draft/msgid=ID3;time=YYYY-MM-DDThh:mm:ss.sssZ :nick!ident@host PRIVMSG target :message
+    [recv] @label=ID3;time=YYYY-MM-DDThh:mm:ss.sssZ;delivered :nick!ident@host TAGMSG target
+    [send] :irc.host BATCH -ID
+    [recv] @label=ID1;time=YYYY-MM-DDThh:mm:ss.sssZ;read :nick!ident@host TAGMSG target
+    [recv] @label=ID2;time=YYYY-MM-DDThh:mm:ss.sssZ;read :nick!ident@host TAGMSG target
+    [recv] @label=ID3;time=YYYY-MM-DDThh:mm:ss.sssZ;read :nick!ident@host TAGMSG target
+
+Sender cancels message and clears the text-input field
+
+    [send] @time=YYYY-MM-DDThh:mm:ss.sssZ;@typing=false :nick!ident@host TAGMSG target
+
+## Use Cases
+This specification is intended for use on servers and/or channels where tracking message status would be beneficial. Current implementations on similar platforms has proven useful, especially in collaborative team environments. The ability to send delivery receipts can also prove useful when connections are unstable and some messages may be dropped prior to reaching their intended target.
+
+## Limitations
+Clients must support the [`message-tags`][tags] and [`server-time`][time] capabilities. [`labeled-response`][labels] is also required for `delivered` and `read` tags. Specific [Privacy Considerations](#privacy-considerations) are addressed below.
+
+This specification is primarily intended for instances where all users are familiar with each other. In large channels of unknown users, the additional client tags add extra data that must be transmitted over the network with little potentential benefit to the users.
+
+## Privacy Considerations
+Read receipts introduce additional privacy concerns for users who may not wish to inform others of when they have read their messages. Servers SHOULD implement mechanisms at all levels to allow full control over the ability to send and receive `read` client tags. Network operators SHOULD be able to enable or disable the tag server-wide. Channels SHOULD have mechanism in place to allow operators to prevent this tag channel wide (e.g. a channel mode). Users SHOULD have the option of disabling read receipts across the entire network. IRC services SHOULD provide a setting to disable read receipts for an account.
+
```

Comment:
A lot of these SHOULDs are non-normative implementation considerations, and are covered by the 
[Moderation considerations section](http://ircv3.net/specs/core/message-tags-3.3.html#moderation-considerations) of the `message-tags` spec.